src/utils/date.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Converts a date string to a timestamp.
+ * @param { Date } dateField - The date string in the format "YYYY-MM-DD" or a full ISO
+ * datetime string.
```

Comment:
Update the comment here based on the changes above. Let's also wrap it properly (max 80 characters in line):

```suggestion
* @param { Date } dateField - The date string in the format "YYYY-MM-DD" or a
 * full ISO datetime string.
```

**Quick tip:** if you use Visual Studio Code: You can set the indicator at 80th column by adding this configuration to your Visual Code `setting.json` file:
```
  "editor.rulers": [80],
```

Result:

<img width="718" alt="image" src="https://github.com/threshold-network/website/assets/40306834/239a616b-258c-4263-9658-5d53158c8292">

This way you can see when your comment goes past 80 characters (in one line) limit

Reply:
Addressed in https://github.com/threshold-network/website/pull/94/commits/0f0f64c28a4043752e800d86acdacc80abb9fa74